ui/app/components/clients/dashboard.js

```diff
@@ -24,40 +24,56 @@ export default class Dashboard extends Component {
     { key: 'entity_clients', label: 'entity clients' },
     { key: 'non_entity_clients', label: 'non-entity clients' },
   ];
-  // TODO remove this adapter variable? or set to /clients/activity ?
-  adapter = this.store.adapterFor('clients/new-init-activity');
+
+  // needed for startTime modal picker
+  months = Array.from({ length: 12 }, (item, i) => {
+    return new Date(0, i).toLocaleString('en-US', { month: 'long' });
+  });
+  years = Array.from({ length: 5 }, (item, i) => {
+    return new Date().getFullYear() - i;
+  });
 
   @service store;
 
   @tracked barChartSelection = false;
   @tracked isEditStartMonthOpen = false;
-  @tracked startTime = this.args.model.startTime;
-  @tracked endTime = this.args.model.endTime;
+  @tracked responseRangeDiffMessage = null;
+  @tracked startTimeRequested = null;
+  @tracked startTimeFromResponse = this.args.model.startTimeFromLicense; // ex: ['2021', 3] is April 2021 (0 indexed)
+  @tracked endTimeFromResponse = this.args.model.endTimeFromLicense;
   @tracked startMonth = null;
   @tracked startYear = null;
   @tracked selectedNamespace = null;
   // @tracked selectedNamespace = 'namespacelonglonglong4/'; // for testing namespace selection view
 
   get startTimeDisplay() {
-    if (!this.startTime) {
+    // changes 3,2021 to March, 2021
+    if (!this.startTimeFromResponse) {
       // otherwise will return date of new Date(null)
       return null;
     }
-    let formattedAsDate = new Date(this.startTime); // on init it's formatted as a Date object, but when modified by modal it's formatted as RFC3339
-    return format(formattedAsDate, 'MMMM yyyy');
+    // ['2021, 2'] is March 2021
```

Comment:
might be helpful to clarify comment as:  `// this.startTimeFromResponse  = ['2021', 2] is March 2021`
(Helpful to know the year is a string and month is an integer